docs(Kommander): rename cloud providers to infrastructure providers (#2991)
mesosphere/dcos-docs-site: pages/ksphere/kommander/1.1.0-beta/operations/infrastructure-providers/configure-aws-infrastructure-provider-roles/index.md

---
layout: layout.pug

> navigationTitle: Configure an AWS Provider with a User Role

Reviewer: Was "Infrastructure" forgotten here or left out on purpose?

}
```

To use the role created, attach the following policy to the role which is already attached to your Kommander cluster.

> Replace `YOURACCOUNTRESTRICTION` with the AWS Accoount ID where the role you like to `AssumeRole` is saved. Also, replace `THEROLEYOUCREATED` with the AWS Role name.

Reviewer: I think "Accoount" should be "Account".

}
```

#### Create role when creating the management cluster

> Instead of doing this manually you can create the following file, name it, `iam-stsAssumeKommander.tf`, into the `extras/provisioner/` directory next to your `cluster.yaml` file. This will create the role when you create or update your cluster with `konvoy up`. Ensure you replace `THEROLEYOUCREATED` with the AWS Role name.

Reviewer 2: ```suggestion
Instead of doing this manually create the following file, `iam-stsAssumeKommander.tf`, int the `extras/provisioner/` directory, next to your `cluster.yaml` file. This creates the role when you create or update your cluster with `konvoy up`. Ensure you replace `THEROLEYOUCREATED` with the AWS Role name.
```

Author: `int` ? do you mean `in`

}
```

### Create Infrastructure Provider

In Kommander, select the Workspace associated with the credentials you are adding.

> Navigate to **Administration > Infrastructure Providers** and click the **Add Infrastructure Provider** button.

Reviewer 2: ```suggestion
Go to  **Administration > Infrastructure Providers** and select the **Add Infrastructure Provider** button.
```

![Adding an Infrastructure Provider](/ksphere/kommander/1.1.0-beta/img/empty-infrastructure-providers.png)
<br />_Adding an Infrastructure Provider_

![Add Infrastructure Provider Form](/ksphere/kommander/1.1.0-beta/img/add-infrastructure-provider.png)
<br />_Add Infrastructure Provider Form_

- Select the Amazon Web Services (AWS) option from the Add Infrastructure Provider.
- Ensure "Role" is selected as the Authentication Method.

> - Enter a name for your infrastructure provider. Consider choosing a name that matches the AWS user.

Reviewer 2: ```suggestion
- Enter a name for your infrastructure provider. Select a name that matches the AWS user.
```

Reviewer: Selecting a name that matches the AWS user is a recommendation, not a requirement as far as I know.

- Enter the Role ARN.
- You can add an External ID if you share the Role with a 3rd party. External IDs secure your environment from accidentally used roles. [Here you can read more about External IDs][external_id].

![AWS Role Infrastructure Provider Form](/ksphere/kommander/1.1.0-beta/img/infrastructure-provider-form-aws-role.png)
<br />_AWS Role Infrastructure Provider Form_

> After the provider is created, its display name or role can be updated.

Reviewer 2: ```suggestion
After the provider is created, the display name or role can be updated.
```
